logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -15,6 +15,21 @@ def close
   end
 end
 
+class DummyInputGenerator < LogStash::Inputs::Base
```

Comment:
I know is not obvious, is in the file I shared, :+1: I understand. I think we should have only 1 way if possible, right? what do you think?